src/components/Text/Text.module.css

```diff
@@ -92,3 +92,156 @@
 .text--2x-spacing {
   margin-bottom: var(--eds-size-2);
 }
+
+/**
+ * 1) A passage of text, including various components (i.e. article, blog post)
+ */
+.text-passage {
+  /**
+	 * Paragraph within text passage
+	 */
+  p {
```

Comment:
Noticed the original `TextPassage` component was not removing the browser spacing styles for `p` and `h1` - `h6` (not using @mixin reset). Is this intentional?